src/layman/util.py

```diff
@@ -208,3 +208,42 @@ def url_for(endpoint, **values):
     else:
         result = flask_url_for(endpoint, **values, _external=True)
     return result
+
+
+def get_publication_info(workspace, publ_type, publ_name, context=None):
```

Comment:
Myšlenky k zamyšlení:
- Nebudou některé další util.* metody velmi podobné? Pak by možná stálo za znovuvyužití.
- Někdy píšeme `context = context or {}` a někde jinde `if not read_access_users: read_access_users = ...`, přestože to má stejný význam.
- Můžeme dokonce napsat: `read_access_users = result.get('access_rights', {}).get('read', publications.get_publication_infos(workspace, publ_type).get((workspace, publ_type, publ_name)            )['access_rights']['read']) ` - určitě s lepším formátováním
- Aktuálně si vynucujeme puštění publications.get_publication_infos. Souhlasím s upřednostněním explicitního puštění, než to vynucovat v sources.